### PR TITLE
Nunjucks engine support added

### PR DESCRIPTION
**Pull request by monikkacha**
### Changes 
- Nunjucks engine added 
- README.md updated 
- `--renderer nunjucks` flag added

### CODE REVIEW

File: src/lib/renderers/nunjucks.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  for (let i = 0; i < limit; i++) {
+    const contact = contactsArray[i];
+    const email = nunjucks.renderString(template, contact);
+    emails.push({
+      to: contact.email,
+      subject: nunjucks.renderString(
+        "Hello, {{ first_name }} {{ subject }}",
+        contact
+      ),
```
Comment (charlesyu108) [inline src/lib/renderers/nunjucks.ts:46]: Let's just make this `{{ subject }}` for simplicity

**Review by charlesyu108 — CHANGES_REQUESTED** [review]
One small change, but otherwise great job @monikkacha! Pulled it in and looks nice. Thanks for getting this thru, this is awesome 💯 💯 

Once you make the fixes and we merge it in, I can push a publish a new release.

File: src/lib/renderers/nunjucks.ts (hunk 1 above)
Comment (monikkacha) [reply src/lib/renderers/nunjucks.ts]: resolved

Comment (charlesyu108) [reply src/lib/renderers/nunjucks.ts]: Sorry, i was not clear. I meant let's just have the subject be simply `{{ subject }}` with no "Hello,"

Comment (monikkacha) [reply src/lib/renderers/nunjucks.ts]: Hello, Removed .

**Review by charlesyu108 — APPROVED** [review]

**Pull request merged**